scripts/functional-tests.sh

```diff
@@ -317,8 +320,8 @@ test_model_train() {
     if [[ "$(uname)" == Linux ]]; then
         # real `ilab model train` on Linux produces models/ggml-model-f16.gguf
         # fake gml-model-f16.gguf the same as merlinite-7b-lab-Q4_K_M.gguf
-        test -f "${ILAB_DATA_DIR}/models/ggml-model-f16.gguf" || \
```

Comment:
(No action required) nit: You could put the full path in a variable (`${ILAB_DATA_DIR}/models/ggml-model-f16.gguf`), then you wouldn't need to construct it twice here.

Reply:
Done!